src/sfctl/custom_compose.py

```diff
@@ -64,18 +64,18 @@ def create_app_health_policy(
     )
 
```

Comment:
Since this is a breaking API change update the `history.rst` file, and increment the `setup.py` version by one major version.

Reply:
Where is history.rst file?
For the version, are you suggesting 3.0? That seems unreasonable for just a naming change. Are we considering a minor version change?

Reply:
Ah sorry wrong file name, it's `src/README.rst`. And yeah, we should do `3.0.0`. We're really trying to follow semantic versioning here and since it is an API change (parameter rename), it needs to be a major version.